packages/manager/apps/ips/src/pages/order/sections/OrganisationSection.component.tsx

```diff
@@ -25,37 +31,58 @@ export const OrganisationSection: React.FC = () => {
   const { t } = useTranslation('order');
   const { organisations, isLoading } = useGetOrganisationsList();
 
+  const noOrganisationInfoLabel = useMemo(() => {
+    if (selectedOrganisation) {
+      return t('organisation_selection_info');
+    }
+    if (!selectedOrganisation && isRegionInEu(selectedRegion)) {
+      return t('organisation_selection_description_no_organisation_RIPE');
+    }
+    if (!selectedOrganisation && !isRegionInEu(selectedRegion)) {
+      return t('organisation_selection_description_no_organisation_ARIN');
+    }
+    return undefined;
```

Comment:
Is it really possible to not have a value ?


```suggestion
    return isRegionInEu(selectedRegion))
      ? t('organisation_selection_description_no_organisation_RIPE')
      : t('organisation_selection_description_no_organisation_ARIN');
```